### PR TITLE
feat: Add UntypedAbstractActorWithTimers

### PR DESCRIPTION
**Pull request by He-Pin**
Motivation:
refs: https://github.com/apache/pekko/issues/1360

Result:
Easy usage in Java

### CODE REVIEW

**Comment by laglangyue** [comment]
looks good

File: docs/src/test/java/jdocs/testkit/TestKitDocTest.java [hunk 1]
```diff
@@ -31,15 +32,6 @@
...
 import com.typesafe.config.ConfigFactory;
 
-import org.apache.pekko.actor.AbstractActorWithTimers;
-import org.apache.pekko.actor.ActorKilledException;
-import org.apache.pekko.actor.ActorRef;
-import org.apache.pekko.actor.ActorSystem;
-import org.apache.pekko.actor.Kill;
-import org.apache.pekko.actor.PoisonPill;
-import org.apache.pekko.actor.Props;
-import org.apache.pekko.actor.Terminated;
```
Comment (pjfanning) [inline docs/src/test/java/jdocs/testkit/TestKitDocTest.java:41]: Could we keep these imports? This class is part of our docs and it useful to be explicit in the imports.

File: docs/src/test/java/jdocs/testkit/TestKitDocTest.java [hunk 2]
```diff
@@ -107,6 +99,27 @@ void triggerScheduling() {
       getTimers().startSingleTimer(SCHED_KEY, new ScheduledMessage(), Duration.ofMillis(500));
     }
   }
+
+  static class TestTimerUntypedActor extends UntypedAbstractActorWithTimers {
+    private static Object SCHED_KEY = "SchedKey";
```
[the remark this answers is not in the document]
Comment (pjfanning) [reply docs/src/test/java/jdocs/testkit/TestKitDocTest.java:104]: `static final String` would be my preference

**Review by Roiocam — APPROVED** [review]
LGTM

**Pull request merged**